src/Scene/Quadtree.js

```diff
@@ -50,6 +50,9 @@ define('Scene/Quadtree', [
         this.tileType = type;
         this.minLevel = 2;
         this.maxLevel = 17;
+        this.wgs84TileLayer = null;
+        this.colorLayers = [];
+        this.elevationLayers = [];
```

Comment:
I'm going to push another commit that moves layer knowledge and configuration elsewhere.